packages/app/src/cli/services/dev/ui.tsx

```diff
@@ -92,6 +93,15 @@ export async function renderDev({
   }
 }
 
+export async function renderReplay({selectedRun, abortController, app, extension}: ReplayProps) {
+  return render(
+    <Replay selectedRun={selectedRun} abortController={abortController} app={app} extension={extension} />,
+    {
+      exitOnCtrlC: false,
+    },
+  )
+}
+
```

Comment:
This seems like it shouldn't be in the `dev` folder since it's unrelated to that command. Do we need a new replay folder?

Reply:
Yup makes sense! I'll move it over 👍